recipes/pyknos/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyknos" %}
+{% set version = "0.16.0" %}
+{% set python_min = "3.8" %}
```

Comment:
I acknowledge and understand that this is outside of the supported global pins of Python 3.9 to Python 3.12, and am selecting this intentionally to match lower bounds of the PyPI distributions this package is meant to work with.